packages/website/docs/Slider.md

```diff
@@ -1,16 +1,16 @@
 # Best Practices
 
-A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect such as a percentage of opacity. They are also used when an approximate value is sufficient, for example when choosing a flight departure time.
+A slider is appropriate when providing **an exact value is not important**. Sliders typically allow users to adjust the intensity of an effect, such as a percentage of opacity or when an approximate value is sufficient, for example when choosing a flight departure time. It helps visualize a range of allowed values. 
 
-A slider visualizes the range of allowed values. Use a slider when page real estate is not an issue and when visualizing the range helps users make more informed choices.
-
-The slider should have a range of up to 100 values. If your range is very short (i.e., 5 values or less) or very large (i.e., over 50 values), consider using a [numeric input](#/form/NumericInput) instead.
+* Use a slider when page real estate is not an issue and when visualizing the range helps users make more informed choices.
+* The slider should have a range of up to 100 values. 
+* If the range is short (i.e., 5 values or less) or large (i.e., over 50 values), consider using a [numeric input](https://plasma.coveo.com/form/NumericInput) instead.
```

Comment:
Why the bullet points?

Reply:
I felt like it was easier to digest the use cases as a list

Reply:
Ah, ok. To me it's just best practices like the paragraph above, that's why I don't see the point. Should we introduce them with a sentence that ends with a colon, then?